sonic-ycabled/ycable/ycable_utilities/y_cable_helper.py

```diff
@@ -2665,7 +2666,7 @@ def handle_show_ber_cmd_arg_tbl_notification(fvp, xcvrd_show_ber_cmd_arg_tbl, xc
                     helper_logger.log_warning("Failed to execute cli cmd get_alive_status API for port {} due to {}".format(physical_port,repr(e)))
             if res is not None:
                 fvs_log = swsscommon.FieldValuePairs(
-                    [("cable", str(res))])
```

Comment:
without this fix what was the impact?

Reply:
This will just result in executing the show CLI
$show mux cablealivestatus as
```PORT       ATTR    HEALTH
---------  ------  --------
 Ethernet0  cable True```

I think preferred way would be 
```PORT       ATTR    HEALTH
---------  ------  --------
Ethernet0 cable_alive True```